inc/policydeployfile.class.php

```diff
@@ -281,29 +281,37 @@ public function showValueInput($value = '', $itemType = '', $itemId = 0) {
          $destination_base = '';
          $destination = '';
       }
-      $out = PluginFlyvemdmFile::dropdown([
+      $filesDropdown = PluginFlyvemdmFile::dropdown([
          'display'   => false,
          'name'      => 'items_id',
          'value'     => $itemId,
       ]);
-      $out .= '<br>';
-      $out .= __('copy to', 'flyvemdm');
-      $out .= '<br>';
+
+      //Copy to
       $path = new PluginFlyvemdmWellknownpath();
       $path->getFromDBByPath($destination_base);
-      $out .= PluginFlyvemdmWellknownpath::dropdown([
+
+      $copyDropdown = PluginFlyvemdmWellknownpath::dropdown([
          'display'   => false,
          'name'      => 'destination_base',
          'value'     => $path->getID(),
       ]);
-      $out .= '<input type="text" name="value[destination]" value="' . $destination . '" />';
-      $out .= '<br>';
-      $out .= __('Remove when the policy is removed', 'flyvemdm');
-      $out .= "&nbsp;&nbsp;" . Dropdown::showYesNo('value[remove_on_delete]', $removeOnDelete, -1, ['display' => false]);
-      //$out .= '<input type="hidden" name="value[remove_on_delete]" value="' . $removeOnDelete . '" />';
-      $out .= '<input type="hidden" name="itemtype" value="' . $itemtype . '" />';
-
-      return $out;
+
+      $removeDropdown = Dropdown::showYesNo('value[remove_on_delete]', $removeOnDelete, 
+                                                -1, ['display' => false]);
+
+      $data = [
+            'files'         => [
+                  'dropdown'        => $filesDropdown,
+                  'itemtype'        => $itemtype,
+                  'copy'            => $copyDropdown,
```

Comment:
Please be more semantic and change the text copy and the var $copyDropdown to knownPath